composer.json

```diff
@@ -30,6 +30,7 @@
         "drupal/devel": "^5.1",
         "drupal/diff": "^1.0",
         "drupal/easy_breadcrumb": "^2.0",
+        "drupal/edit_media_modal": "^1.0",
```

Comment:
They released version 2.0 with CKEditor5 support the day you submitted your patch. It doesn't look like they included your patch in the release, so I'll leave it up to you if you want to update to v2 and reroll your patch for that version.